config/hardware/addons/accelerometers/adxl345_usb.cfg

```diff
@@ -0,0 +1,17 @@
+[mcu rpi]
```

Comment:
What is the difference between this file and the `adxl345_rpi.cfg` file ?

Reply:
XD, No difference I pushed the wrong file. :-(
Let me know if the new file suits.